lib/tasks/fix_fdw_credentials.rake

```diff
@@ -0,0 +1,35 @@
+desc 'Change FDW server and user credentails accroding to dot env file'
+task fix_fdw_credentials: :environment do
+  fix_primary_server_fdw_credentials
+  fix_remote_server_fdw_credentials
+end
+
+def fix_primary_server_fdw_credentials
+  config = ActiveRecord::Base.configurations['secondbase'][Rails.env]
+  server = 'fis'
+  active_record_execute(alter_fdw_server_query(server, config))
+  active_record_execute(alter_fdw_user_query(server, config))
+end
+
+def fix_remote_server_fdw_credentials
+  config = ActiveRecord::Base.configurations[Rails.env]
+  server = 'openhub'
+  second_base_execute(alter_fdw_server_query(server, config))
+  second_base_execute(alter_fdw_user_query(server, config))
+end
+
+def alter_fdw_server_query(server_name, config)
+  "ALTER SERVER #{server_name} OPTIONS(set host '#{config['host']}', set dbname '#{config['database']}', set port '#{config['port']}')"
+end
+
+def alter_fdw_user_query(server_name, config)
+  "ALTER USER MAPPING FOR #{config['username']} SERVER #{server_name} OPTIONS(SET password '#{config['password']}')"
```

Comment:
Do the folks at Imaginea have a local  copy of the development database?  If so, what data is in it?  For Ohloh-UI, it is no longer out of the question to have a local development DB with it being "only" 25 GB.  But we still need the FIS DB and that is still around 1.5 TB. Not an option.

Therefore we in Burlington use the development database on the oh-db01.dc1.lan server.

Therefore, we are assuming that the development (and staging) databases are created by the weekly restore process and we will only ever have test databases for Ohloh and FIS locally.

And therefore, we don't need to create the development database.

Finally, I'm not sure this approach changes anything. The issue seems to be that, in the test environment, the test framework immediately load and processes factory files before the rake task is invoked.

It's almost as if we need to modify values in the structure file before `rake db:structure:load`